File: EjerciciosC++/conejosVenta/conejosVenta/conejos.cpp
#include <iostream>
using namespace std;

int main() {

	int conejosBlancos, conejosNegros;
	float precioBlancos, precioNegros;

	cout << "Introduzca el precio de un conejo blanco: ";
	cin >> precioBlancos;
	cout << "\nIntroduzca el precio de un conejo negro: ";
	cin >> precioNegros;
	cout << "\nIntroduzca la cantidad de conejos blancos que desea comprar: ";
	cin >> conejosBlancos;
	cout << "\nIntroduzca la cantidad de conejos negros que desea comprar: ";
	cin >> conejosNegros;

	//Verificacion de datos:
	//	Todas las variables debe ser mayores o iguales a cero

	if (conejosBlancos < 0 || conejosNegros < 0 || precioBlancos < 0 || precioNegros < 0) {

		cout << "\nLos datos ingresados no son validos" << endl;
	
	}

	else if(conejosBlancos == conejosNegros){

		cout << "\n\nLa cantidad de conejos blancos es igual a la de conejos negros";

	}
	else if (conejosBlancos > conejosNegros) {

		cout << "\n\nConejos mas vendidos: Blancos" << endl;

	}
	else {

		cout << "\n\nConejos mas vendidos: Negros" << endl;

	}

	cout << "\nTotal de conejos vendidos: " << conejosBlancos + conejosNegros << endl;
	cout << "Total de la venta: " << conejosBlancos * precioBlancos + conejosNegros * precioNegros << endl;

		
	system("PAUSE");
	return 0;
}

File: Ejercicios C#/Conejos/Conejos/conejos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Conejos
{
    class Conejos
    {
        static void Main()
        {
            int conejosBlancos, conejosNegros;
            float precioBlancos, precioNegros;

            Console.Write("Escriba el precio de un conejo Blanco: ");
            precioBlancos = float.Parse(Console.ReadLine());

            Console.Write("Escriba el precio de un conejo Negro: ");
            precioNegros = float.Parse(Console.ReadLine());

            Console.Write("Escriba la cantidad de conejos Blancos: ");
            conejosBlancos = int.Parse(Console.ReadLine());

            Console.Write("Escriba la cantidad de conejos Negros: ");
            conejosNegros = int.Parse(Console.ReadLine());


            //Si alguno es negativo no se realizan los cálculos

            if(conejosNegros < 0 || conejosBlancos < 0 || precioBlancos < 0 || precioNegros < 0)
            {

                Console.WriteLine("\n\nLos datos son invalidos");

            }
            else
            {

                if (conejosBlancos == conejosNegros)
                {

                    Console.WriteLine("\nConejos mas vendidos: Iguales");

                }else if (conejosBlancos < conejosNegros)
                {

                    Console.WriteLine("\nConejos mas vendidos: Negros");

                }
                else
                {

                    Console.WriteLine("\nConejos mas vendidos: Blancos");

                }


                Console.WriteLine("Total de conejos vendidos: {0}", conejosNegros + conejosBlancos);
                Console.WriteLine("Total de la venta: {0:F3}", conejosBlancos * precioBlancos + conejosNegros * precioNegros);


                Console.Write("Presione una tecla para continuar ...");
                Console.Read();
            }

        }
    }
}
